test/integration/install_test.go

```diff
@@ -397,7 +397,7 @@ func TestCheckHelmStableBeforeUpgrade(t *testing.T) {
 		t.Skip("Skipping as this is not a helm upgrade test")
 	}
 
-	testCheckCommand(t, "", TestHelper.UpgradeHelmFromVersion(), "", TestHelper.UpgradeHelmFromVersion())
+	testCheckCommand(t, "", TestHelper.UpgradeHelmFromVersion(), "", TestHelper.UpgradeHelmFromVersion(), false)
```

Comment:
I see this is what we did when the add-on checks were introduced in the past, but given this new parameter is only set to false here, I think it might be clearer that instead of modifying the `testCheckCommand` function along with all its calling points, we just comment out this call here, with a to-do saying this should be uncommented when the next stable after 2.8.1 comes out.

Reply:
@Pothulapati My suggestion was to comment out this `testCheckCommand()` call because it's a no-op in this case, and so not having to modify its definition nor the other places where it's called from.

Reply:
@alpeb but that would mean we would be totally skipping the check tests here (which can fail). My thinking with the `compareOutput` flag is that we would only not check the output but atleast have a way to check if the command failed.

Reply:
Currently, if `compareOutput` is false then `testCheckCommand()` doesn't do anything besides initializing some vars.